typeshed/__init__.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+
+__all__ = ['__location__', '__version__']
+
+
+try:
+    import pkg_resources
+except ImportError:
+    __location__ = os.path.dirname(os.path.dirname(__file__))
+else:
+    __location__ = pkg_resources.resource_filename("typeshed", "")
+__version__ = "18.4.0"
+
+
+# Deprecated name kept for backwards compatibility.
+typeshed = __location__
```

Comment:
Is this just for compatibility with the antiquated typeshed package currently on PyPI? Isn't it better to just get rid of it then?

Reply:
Yeah, I didn't want to break whatever usage is there in the wild but you're probably right that there might not be any.